src/Fixer/Basic/BracesFixer.php

```diff
@@ -354,13 +354,29 @@ function ($item) {
                     continue;
                 }
 
-                if (1 === $nestLevel && $nestToken->equalsAny([';', '}'])) {
+                if (1 === $nestLevel && $nestToken->equalsAny([';', '}', [T_COMMENT]])) {
```

Comment:
I suggest changing this to:
>if (1 === $nestLevel && ($nestToken->equalsAny([';', '}']) || $nestToken->isComment())) {

as it would support this test case (please add it as well)

```php
            [
                '<?php
if (1==1) {
    $a = 1;
    /** @var int $b */
    $b = a();
}',
                '<?php
if (1==1) {
    $a = 1;
    /** @var int $b */
$b = a();
}'
,
            ]
```